src/core/manager.c

```diff
@@ -1803,6 +1803,9 @@ Manager* manager_free(Manager *m) {
         free(m->switch_root);
         free(m->switch_root_init);
 
+        sd_bus_track_unref(m->subscribed);
+        strv_free(m->deserialized_subscribed);
```

Comment:
I think it'd prefer to rename this field. (`inherited_subscribed` or `previously_subscribed` or something.)  I think it'll be confusing if it's called "deserialized", but actually it's set at runtime.

Reply:
Well, it _is_ serialized. This is just the already existing field from before additionally used in the "bus disconnect" case — I think I'd prefer not to change the name.

I added these two lines because these fields actually were just not previously freed in manager free, which seems like an accidental omission to me either way. However, this did not previously cause any leak afaict because no code path reached manager free before freeing these fields. With these changes, it potentially could, so they must now be freed here for correctness.

Reply:
i agree with @keszybz please rename this, this is too confusing. maybe renamed it "subscribe_serialization" or so?